python/pyspark/ml/clustering.py

```diff
@@ -310,6 +311,374 @@ def _create_model(self, java_model):
         return BisectingKMeansModel(java_model)
 
 
+class LDAModel(JavaModel):
+    """
+    A clustering model derived from the LDA method.
+
+    .. versionadded:: 2.0.0
+    """
+
+    @since("2.0.0")
+    def isDistributed(self):
+        """Indicates whether this instance is of type DistributedLDAModel"""
+        return self._call_java("isDistributed")
+
+    @since("2.0.0")
+    def vocabSize(self):
+        """Vocabulary size (number of terms or terms in the vocabulary)"""
```

Comment:
"terms or terms" must be a mistake from a search-and-replace.  I bet it's supposed to be "terms or words"
Could you fix that here and in the Scala doc too please?